sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/models/QueueMessageItem.java

```diff
@@ -46,7 +46,7 @@
      * The time that the message will again become visible in the Queue.
```

Comment:
This is auto-generated. It'll be stomped over next time the file is regenerated.